### ISSUE TITLE
[BUG] YACReader not updated to latest version

### ISSUE THREAD

**Issue by peligwe** [xthursdayx/yacreaderlibrary-server-docker#62]
I just updated the docker to latest version and YACReader is still version 9.13.1.  

## Expected Behavior
Android app keeps saying server needs to be upgrade to version 9.14.1 or later.

## Steps to Reproduce
root@hme-omvsvr:/# docker exec yacreader YACReaderLibraryServer -v
YACReaderLibraryServer 9.13.1

## Screenshots
<img width="600" alt="image" src="https://github.com/xthursdayx/yacreaderlibrary-server-docker/assets/28747038/1e693ecd-6f3f-40fd-a8ff-f9965be0ce68">

## Command used to create docker container (run/create/compose/screenshot)
<img width="692" alt="image" src="https://github.com/xthursdayx/yacreaderlibrary-server-docker/assets/28747038/afac8b51-f046-4218-9d44-3d9dd8abf1fb">

## Docker logs
[migrations] started
[migrations] no migrations found
----------------------------------------------------------
   _  __________  ____  ______  _____ ____  _____  ___  __
  | |/ /_  __/ / / / / / / __ \/ ___// __ \/   \ \/ / |/ /
  |   / / / / /_/ / / / / /_/ /\__ \/ / / / /| |\  /|   /
 /   | / / / __  / /_/ / _, _/___/ / /_/ / ___ |/ //   |
/_/|_|/_/ /_/ /_/\____/_/ |_|/____/_____/_/  |_/_//_/|_|
Brought to you by xthursdayx
----------------------------------------------------------
GID/UID
----------------------------------------------------------
User uid:    998
User gid:    100
-------------------------------------
[custom-init] No custom files found, skipping...
INFO  2024-02-09T09:51:31.462 YACReaderLibraryServer attempting to start 
INFO  2024-02-09T09:51:31.462 ---------- System & configuration ---------- 
INFO  2024-02-09T09:51:31.463 OS: "Ubuntu 22.04.2 LTS" Version:  "22.04" 
INFO  2024-02-09T09:51:31.463 Kernel: "linux" "6.1.0-0.deb11.13-amd64" Architecture: "x86_64" 
INFO  2024-02-09T09:51:31.463 Libraries:  QMap() 
INFO  2024-02-09T09:51:31.463 -------------------------------------------- 
INFO  2024-02-09T09:51:31.463 YACReaderLibrary starting 
INFO  2024-02-09T09:51:31.464 Running on "172.17.0.14:8080" 
 
INFO  2024-02-09T09:51:31.464 "███████████████████████████████" 
INFO  2024-02-09T09:51:31.464 "███████████████████████████████" 
INFO  2024-02-09T09:51:31.464 "█████ ▄▄▄▄▄ █ █ ▄▄█ ▄▄▄▄▄ █████" 
INFO  2024-02-09T09:51:31.464 "█████ █   █ █ ▀▄▄▀█ █   █ █████" 
INFO  2024-02-09T09:51:31.464 "█████ █▄▄▄█ █▄ █▄██ █▄▄▄█ █████" 
INFO  2024-02-09T09:51:31.464 "█████▄▄▄▄▄▄▄█ █▄█▄█▄▄▄▄▄▄▄█████" 
INFO  2024-02-09T09:51:31.464 "██████▄ ▀ █▄▄█▀▀▀▄█ █▄ ▄ ▄█████" 
INFO  2024-02-09T09:51:31.464 "█████▀▀▄▄██▄▀█▄ ▄ ▀▄█  ▄▄ █████" 
INFO  2024-02-09T09:51:31.464 "█████▄▄██▄█▄▄ █▀▀▀█▄▀ ▀ ▀▄█████" 
INFO  2024-02-09T09:51:31.465 "█████ ▄▄▄▄▄ █▄▄█▄▀  ▄ ▀▄▀▄█████" 
INFO  2024-02-09T09:51:31.465 "█████ █   █ █▄▀█ ▄▀█▄▄█ ▀██████" 
INFO  2024-02-09T09:51:31.465 "█████ █▄▄▄█ █ ▀▀▀▄▀ █  ▄▀ █████" 
INFO  2024-02-09T09:51:31.465 "█████▄▄▄▄▄▄▄███▄▄▄▄███▄██▄█████" 
INFO  2024-02-09T09:51:31.465 "███████████████████████████████" 
INFO  2024-02-09T09:51:31.465 "███████████████████████████████" 
[ls.io-init] done.
WARN  2024-02-09T10:45:11.593 QObject: Cannot create children for a parent that is in a different thread.
(Parent is RequestMapper(0x562e7b556b30), parent's thread is QThread(0x562e7b54c490), current thread is QThread(0x562e7b559600)

**Comment by accessdenied56** [comment]
Yeah I'm seeing the same. I note that there is an issue with compiling the 7zip version. I'm using the unarr and according to version 1.7.2 it should be server version 9.14.1

**Comment by xthursdayx** [comment]
This docker image has been undergoing development this week. The new images were just pushed to Docker Hub, which should remedy this issue. Please pull the new image and create a new container and let me know if you're still having this issue. Thanks.

**Comment by peligwe** [comment]
Created new container with the 7zip tag but the library is not created or updating,

[migrations] started
[migrations] no migrations found
----------------------------------------------------------
   _  __________  ____  ______  _____ ____  _____  ___  __
  | |/ /_  __/ / / / / / / __ \/ ___// __ \/   \ \/ / |/ /
  |   / / / / /_/ / / / / /_/ /\__ \/ / / / /| |\  /|   /
 /   | / / / __  / /_/ / _, _/___/ / /_/ / ___ |/ //   |
/_/|_|/_/ /_/ /_/\____/_/ |_|/____/_____/_/  |_/_//_/|_|
Brought to you by xthursdayx
----------------------------------------------------------
GID/UID
----------------------------------------------------------
User uid:    998
User gid:    100
-------------------------------------
[custom-init] No custom files found, skipping...
INFO  2024-02-17T20:32:41.060 YACReaderLibraryServer attempting to start 
INFO  2024-02-17T20:32:41.060 ---------- System & configuration ---------- 
INFO  2024-02-17T20:32:41.061 OS: "Ubuntu 22.04.2 LTS" Version:  "22.04" 
INFO  2024-02-17T20:32:41.061 Kernel: "linux" "6.1.0-0.deb11.13-amd64" Architecture: "x86_64" 
INFO  2024-02-17T20:32:41.061 Libraries:  QList() 
INFO  2024-02-17T20:32:41.061 -------------------------------------------- 
INFO  2024-02-17T20:32:41.061 YACReaderLibrary starting 
INFO  2024-02-17T20:32:41.063 Running on "172.17.0.5:8080" 
 
INFO  2024-02-17T20:32:41.063 "███████████████████████████████" 
INFO  2024-02-17T20:32:41.063 "███████████████████████████████" 
INFO  2024-02-17T20:32:41.063 "█████ ▄▄▄▄▄ ██▄▄▀ █ ▄▄▄▄▄ █████" 
INFO  2024-02-17T20:32:41.063 "█████ █   █ █▀▄██ █ █   █ █████" 
INFO  2024-02-17T20:32:41.064 "█████ █▄▄▄█ █▀█ ▄▄█ █▄▄▄█ █████" 
INFO  2024-02-17T20:32:41.064 "█████▄▄▄▄▄▄▄█ █ █ █▄▄▄▄▄▄▄█████" 
INFO  2024-02-17T20:32:41.064 "█████▀  █▀█▄▀▀   ▀█  ▀▄████████" 
INFO  2024-02-17T20:32:41.064 "██████▀ ▄▀▀▄▀▀█▄▄ ▄ ▀▀█▄▀ █████" 
INFO  2024-02-17T20:32:41.064 "████████▄▄▄▄█▀▄▀  ▄▄█ ▄ ▄▄█████" 
INFO  2024-02-17T20:32:41.064 "█████ ▄▄▄▄▄ ██▄ ▄ ▄▄▀ ▄▄ ▀█████" 
INFO  2024-02-17T20:32:41.064 "█████ █   █ ███ ▄█▄█▀▄  █▄█████" 
INFO  2024-02-17T20:32:41.064 "█████ █▄▄▄█ █ ██▄▄▄▄  █▄▄ █████" 
INFO  2024-02-17T20:32:41.064 "█████▄▄▄▄▄▄▄██▄█▄▄▄▄▄███▄▄█████" 
INFO  2024-02-17T20:32:41.064 "███████████████████████████████" 
INFO  2024-02-17T20:32:41.064 "███████████████████████████████" 
[ls.io-init] done.
WARN  2024-02-17T20:33:43.540 QObject: Cannot create children for a parent that is in a different thread.
(Parent is RequestMapper(0x5648a4ee3790), parent's thread is QThread(0x5648a4ed7450), current thread is QThread(0x5648a4ee6940)
